src/layouts/MainLayout/assets/eventsIcon.vue

```diff
@@ -1,37 +1,29 @@
 <template>
   <svg
     xmlns="http://www.w3.org/2000/svg"
-    width="15.634"
-    height="15.619"
-    viewBox="0 0 15.634 15.619"
+    width="30.101"
+    height="30.071"
+    viewBox="0 0 30.101 30.071"
```

Comment:
I think is better to add the width and height as props and add a default, instead of hardcode here

Reply:
Also is a good idea to include a "className" just in case

Reply:
Thanks for the suggestions, will check